docs/marionette.functions.md

```diff
@@ -355,28 +413,31 @@ var View = Mn.View.extend({
 
   initialize: function() {
     var hash = {
-      'event:one': 'handleEventOne', // This will become a reference to `this.someFn`
-      'event:two': this.someOtherFn
+      'action:one': 'handleActionOne', // This will become a reference to `this.handleActionOne`
+      'action:two': this.handleActionTwo
     };
 
     this.normalizedHash = this.normalizeMethods(hash);
-
-    Mn.bindEvents(this, this, this.normalizedHash);
+    // or equivalent Mn.normalizeMethods(this, hash);
   },
 
-  destroy: function() {
-      Mn.unbindEvents(this, this, this.normalizedHash);
+  do: function(action) {
+    this.normalizedHash[action]();
   },
 
-  handleEventOne: function() {
-    console.log('event:one was fired');
+  handleActionOne: function() {
+    console.log('action:one was fired');
   },
 
-  handleEventTwo: function() {
-    console.log('event:two was fired');
+  handleActionTwo: function() {
+    console.log('action:two was fired');
   }
 
 });
+
+var myView = new MyView();
+myView.do('action:one');
+myView.do('action:two');
 ```
 
 [Live example](https://jsfiddle.net/marionettejs/zzjhm4p1/)
```

Comment:
I just hadn't saved the fiddle on accident.. I _think_ I've updated most of them already.